### ISSUE TITLE
script without `fn main() {}` can't be compiled

### ISSUE THREAD

**Issue by tidyjiang8** [vlang/v#713]
**V version:** v0.1.5. commit id: c83ebe2c6
**OS:** Linux. Ubuntu 18.04
**C compiler version:** gcc version 5.4.0 20160609

**What did you do?**
create a file 'hello.v' with contents (note: no `fn main` here):
```
name := 'Bob'
println(name)
```
and then compile it with cmd `v run hello.v`

**What did you expect to see?**
```
Bob
``` 
**What did you see instead?**
compile error:
```
check types got="void" exp="string"  
pass=1 fn=``
panic: hello.v:2
Fn "println" wrong arg #1. Expected "string" (s)  but got "void"
```
If I put the code snippet to `fn main { }`, no problem; If I put the snippet into `REPL`, still no problem.

**Comment by thisisshub** [comment]
Well I found a solution that you need to mention the datatype of the variable in println line apparently 


hence the now code becomes,
`name := 'Bob'
println(string(name))`

when you execute hello.v now you see the result,
`Bob`

**Comment by christopherallis** [comment]
It seems to be an issue when trying to access variables. 

The following works successfully.
```
age := '3'
println('5')
```

**Comment by tidyjiang8** [comment]
> Well I found a solution that you need to mention the datatype of the variable in `println()` line apparently
> 
> hence the now code becomes,
> 
> ```


> name := 'Bob'
> println(string(name))
> ```
> 
> when you execute hello.v now you see the result,
> `Bob`

@thisisshub

Even I tried this, it still said another issue: 
```println
pass=2 fn=`to_c`
panic: string.v:501
bad token `#` (embedding C code is no longer supported)
```
In fact, the code:
```
name := 'Bob'
println(name)
```
are copied from vlang doc here: https://vlang.io/docs#vars, so I think there must be some bug about it.

**Comment by thisisshub** [comment]
We should request in the documentation too, I guess.

**Comment by medvednikov** [comment]
Fixed in 0.1.8
